plugins/de.cau.cs.kieler.klighd/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: KIELER Lightweight Diagrams (KLighD)
 Bundle-SymbolicName: de.cau.cs.kieler.klighd;singleton:=true
-Bundle-Version: 2.2.1.qualifier
+Bundle-Version: 2.1.1.qualifier
```

Comment:
No, its 2.2.1 now, wrong merge? This seems to be the cause of the build failure as well.